portal.properties.patch

```diff
@@ -51,3 +60,12 @@
  
  # google analytics
  google_analytics_profile_id=
+@@ -224,5 +224,6 @@
+ # Set this to true if you want to check the "hide Putative Passenger Mutations" checkbox by default.
+ # oncoprint.hide_passenger.default=true
+ 
+-# Change this to custom file (e.g. file:/<path>) to have a custom set of genes in query page:
++# Change this to custom JSON file (e.g. file:/<path>) to have a custom set of genes in query page.
++# JSON format should be in format of: https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/components/query/gene_lists.ts
```

Comment:
I think that this is not the right place to add this documentation, if only because it's at least as relevant for people who don't use this Dockerised setup. @sandertan, do you mind if I remove this last hunk?

Reply:
indeed, this documentation should be in portal.properties.EXAMPLE in main repo

Reply:
It should indeed be in portal.properties.EXAMPLE. But it isn't now, so I thought this is a nice temporary solution.

Reply:
@sandertan, could you link to the pull request that adds it to the manual and upstream example file?

Reply:
https://github.com/cBioPortal/cbioportal/pull/3506 @pieterlukasse could you review it?